stone_age/game_board/building_tile.py

```diff
@@ -0,0 +1,83 @@
+from typing import Optional, Union, Iterable
+from stone_age.game_board.simple_types import Player
+from stone_age.simple_types import PlayerOrder
+from stone_age.game_board.interfaces import InterfaceFigureLocationInternal
+from stone_age.game_board import simple_building
+from stone_age.game_board import variable_building
+from stone_age.game_board import arbitrary_building
+from stone_age.simple_types import HasAction, Effect, ActionResult
+
+
+class BuildingTile(InterfaceFigureLocationInternal):
+
+    def __init__(self, cards: Iterable[Union[simple_building.SimpleBuilding,
+        arbitrary_building.ArbitraryBuilding,
+        variable_building.VariableBuilding]]):
+        self.cards: list[Union[simple_building.SimpleBuilding,
+        arbitrary_building.ArbitraryBuilding,
+        variable_building.VariableBuilding]] = list(cards)
+        self.figures: Optional[PlayerOrder] = None
+
+
+    def place_figures(self, player: Player, figure_count: int) -> bool:
+        if self.try_to_place_figures(player, figure_count) != HasAction.NO_ACTION_POSSIBLE:
+            player.player_board.take_figures(1)
+            self.figures = player.player_order
+            return True
+        return False
+
+
+    def try_to_place_figures(self, player: Player, count: int) -> HasAction:
+        assert isinstance(player, Player) and count > 0
+        if count > 1:
+            return HasAction.NO_ACTION_POSSIBLE
+        if not player.player_board.has_figures(count):
+            return HasAction.NO_ACTION_POSSIBLE
+        if self.figures is not None:
+            return HasAction.NO_ACTION_POSSIBLE
+        if not self.cards:
+            return HasAction.NO_ACTION_POSSIBLE
+        return HasAction.AUTOMATIC_ACTION_DONE
```

Comment:
This should return `HasAction.WAITING_FOR_PLAYER_ACTION`, there is no automatic placing of figures in this game.